packages/app/components/pack/PackDetails.tsx

```diff
@@ -68,78 +68,85 @@ export function PackDetails() {
-                              default:
-                                return null;
-                            }
-                          }
-                        }}
-                      />
-                  </>
-                }
-                link={link}
-              />
-            </>
-          )}
+        {!isError && (
+          <View
+            style={[
+              styles.mainContainer,
+              Platform.OS == 'web'
+                ? { minHeight: '100vh', width: '100%' }
+                : { minHeight: Dimensions.get('screen').height },
+            ]}
+          >
+
+            <DetailsComponent
+              type="pack"
+              data={currentPack}
+              isLoading={isLoading}
+              error={error as any}
+              additionalComps={
+                <View>
+                  <FlatList
+                    data={Object.entries(SECTION)}
+                    contentContainerStyle={{ paddingBottom: 50 }}
+                    keyExtractor={([key, val]) => val}
+                    renderItem={({ item }) => {
+                      {
+                        switch (item[1]) {
+                          case SECTION.TABLE:
+                            return (
+                              <TableContainer
+                                currentPack={currentPack}
+                                copy={canCopy}
+                                hasPermissions={isAuthUserPack}
+                              />
+                            );
+                          case SECTION.CTA:
+                            return isAuthUserPack ? (
+                              <AddItemModal
+                                currentPackId={
+                                  currentPackId ? currentPackId : ''
+                                }
+                                currentPack={currentPack}
+                                isAddItemModalOpen={isAddItemModalOpen}
+                                setIsAddItemModalOpen={
+                                  setIsAddItemModalOpen
+                                }
+                                // refetch={refetch}
+                                setRefetch={() =>
+                                  setRefetch((prev) => !prev)
+                                }
+                              />
+                            ) : null;
+                          case SECTION.SCORECARD:
+                            return (
+                              <ScoreContainer
+                                type="pack"
+                                data={currentPack}
+                                isOwner={isOwner}
+                              />
+                            );
+                          case SECTION.CHAT:
+                            return (
+                              <View style={styles.boxStyle}>
+                                <ChatContainer
+                                  itemTypeId={currentPackId}
```

Comment:
There are two ChatContainer components, one here and there is one below as well. you should remove this one since chat opener button supposed to be floating and not have section.